### PR TITLE
[RHPAM-3283] - Align the RHPAM modules with JBoss EAP modules 7.3.x branch

### PR DESCRIPTION
**Pull request by Kaitou786**
See: https://issues.redhat.com/browse/RHPAM-3283

Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by Kaitou786** [comment]
Depends on https://github.com/jboss-container-images/jboss-kie-modules/pull/418

**Review by msmagnanijr — DISMISSED** [review]

File: businesscentral-monitoring/image.yaml [hunk 1]
```diff
@@ -103,36 +111,33 @@ modules:
           - name: jboss.eap.cd.jolokia
           - name: os-eap7-openshift
           - name: jboss.eap.config.openshift
-            version: "16.0-openjdk11"
-          - name: jboss.eap.cd.openshift.modules
-          - name: os-eap7-ping
+            version: "EAP7.3.0-kie"
           - name: os-eap-activemq-rar
             version: "1.1"
           - name: os-eap-launch
```
Comment (spolti) [inline businesscentral-monitoring/image.yaml:117]: version is missing.

File: businesscentral-monitoring/image.yaml [hunk 2]
```diff
@@ -103,36 +111,33 @@ modules:
...
           - name: os-eap7-launch
```
Comment (spolti) [inline businesscentral-monitoring/image.yaml:118]: version is missing.

File: controller/image.yaml [hunk 3]
```diff
@@ -86,36 +94,32 @@ modules:
           - name: jboss.eap.cd.jolokia
           - name: os-eap7-openshift
           - name: jboss.eap.config.openshift
-            version: "16.0-openjdk11"
-          - name: jboss.eap.cd.openshift.modules
-          - name: os-eap7-ping
+            version: "EAP7.3.0-kie"
           - name: os-eap-activemq-rar
             version: "1.1"
-          - name: os-eap-launch
```
Comment (spolti) [inline controller/image.yaml:94]: why removing?
version is missing as well.

File: controller/image.yaml [hunk 4]
```diff
@@ -86,36 +94,32 @@ modules:
...
           - name: os-eap7-launch
```
Comment (spolti) [inline controller/image.yaml:100]: version is missing.

File: kieserver/image.yaml [hunk 5]
```diff
@@ -221,45 +228,43 @@ modules:
...
           - name: jboss.eap.cd.jolokia
           - name: os-eap7-openshift
           - name: jboss.eap.config.openshift
-            version: "16.0-openjdk11"
-          - name: jboss.eap.cd.openshift.modules
-          - name: os-eap7-ping
+            version: "EAP7.3.0-kie"
           - name: os-eap-activemq-rar
             version: "1.1"
           - name: os-eap-launch
```
Comment (spolti) [inline kieserver/image.yaml:244]: version is missing.

File: kieserver/image.yaml [hunk 6]
```diff
@@ -221,45 +228,43 @@ modules:
...
           - name: os-eap7-launch
```
Comment (spolti) [inline kieserver/image.yaml:245]: version is missing.

**Review by spolti — CHANGES_REQUESTED** [review]
nice work, just a few comments.

**Review by spolti — APPROVED** [review]
Awesome, Thanks :)

**Pull request merged**